openweathermap-ios/openweathermap-ios/Weather/Model/WeatherModel.swift

```diff
@@ -0,0 +1,10 @@
+
+struct WeatherModel {
```

Comment:
WeatherModel vs WeatherData can be a little confusing. We should not have to look to the implementation to discover the difference between those structures.

Reply:
The objective of the "WeatherModel" is to have a structure with the most varied climatic conditions. This link shows the different "ids" for each weather condition.

https://openweathermap.org/weather-conditions

I think it makes more sense now
![image](https://user-images.githubusercontent.com/21261476/79125086-ee3e6900-7d73-11ea-9c0c-8c3b58029664.png)

If you are still confused what can be done to improve?

Reply:
I don't know what to do either. Hum... what do you think to add a suffix `Response` to the `WeatherData` to let clear that this structure belongs to the API response.

Reply:
I think this thread is close, right?